### ISSUE TITLE
"30-second intro to OOP" should be moved out of datetime notebook

### ISSUE THREAD

**Issue by brian-rose** [ProjectPythia/pythia-foundations#77]
As discussed in #38, the section in the datetime notebook ["30-second intro to Object Oriented Programming"](https://foundations.projectpythia.org/core/datetime/datetime.html#thirty-second-introduction-to-object-oriented-programming) belongs somewhere else earlier in the Foundations sequence.

To be dealt with later...

**Comment by dcamron** [comment]
If you want to leave this open even after #81 as a reminder to have this material pop up elsewhere, we can.

**Comment by brian-rose** [comment]
Yes, I would leave this open as a reminder to find a new home for the material (and also where to find it in the old commits).

**Comment by brian-rose** [comment]
We need to find a home for this. Perhaps it will fit with "How to build a python package" material.
